Runtime/RosMsgs.cs

```diff
@@ -412,41 +504,159 @@ public class Path : IRosMsg, IStamped
 
 namespace ProBridge.ROS.Msgs.Visualization
 {
+
+    public class UVCoordinate : IRosMsg
+    {
+        string IRosMsg.GetRosType() { return "visualization_msgs.msg.UVCoordinate"; }
+
+        /// <summary>
+        /// Location of the pixel as a ratio of the width of a 2D texture.
+        /// Values should be in range: [0.0-1.0].
+        /// </summary>
```

Comment:
Can be a regular comment, because like this, it will show as a tip for variable u only. Or maybe duplicate it for both.